django_mongodb/compiler.py

```diff
@@ -34,12 +34,19 @@ def __init__(self, *args, **kwargs):
     def _get_group_alias_column(self, expr, annotation_group_idx):
         """Generate a dummy field for use in the ids fields in $group."""
         replacement = None
-        if isinstance(expr, Col):
-            col = expr
+
+        # Unwrap Ref (in this part of the pipeline we can't do references over $projected fields).
+        expr_ = expr
+        while isinstance(expr_, Ref):
```

Comment:
Don't know if there could be a ref of refs. (it could, but Django creates it?)

Reply:
I think we've triaged pretty much all of the complicated queries by now, so if there's no test for it, I wouldn't worry about it.